setup.cfg

```diff
@@ -0,0 +1,4 @@
+[flake8]
```

Comment:
flake8 is great, but you might go with [ruff](https://github.com/astral-sh/ruff) instead. It can be made to run the same rules and it can also fix things for you.

Reply:
I know I need to move things to ruff, but this is a longer term thing.  I should do it as part of my big refactor.